code/game/objects/items/weapons/space_harpoon.dm

```diff
@@ -67,14 +79,16 @@
 		teleport(AtomTurf, UserTurf)
 
 /obj/item/weapon/bluespace_harpoon/proc/teleport(var/turf/source, var/turf/target)
-	for(var/atom/movable/AM in source)
-		if(istype(AM, /mob/shadow))
-			continue
-		if(!AM.anchored)
-			if(prob(offset_chance))
-				AM.forceMove(get_turf(pick(orange(teleport_offset,source))))
-			else
-				AM.forceMove(target)
+	if(Charging == 0)
```

Comment:
Same as above - TRUE or FALSE.

Reply:
!Charging works best because if/inside code only runs if the Charging is set to TRUE. That's same as Charging == 0

Reply:
Or this, yeah.